cmd/main.go

```diff
@@ -28,26 +43,85 @@ func main() {
 			Name:  "log.debug",
 			Usage: "[optional] Log debug info",
 		},
-	}
-
-	app.Commands = []cli.Command{
-		remote.Command(),
-		agent.Command(),
+		cli.StringFlag{
+			Name:  "listen-address",
+			Usage: "[optional] Address to listening",
+			Value: ":9090",
+		},
+		cli.StringFlag{
+			Name:  "proxy-url",
+			Usage: "[optional] URL to proxy",
+			Value: "http://localhost:9999",
+		},
+		cli.DurationFlag{
+			Name:  "read-timeout",
+			Usage: "[optional] Maximum duration before timing out read of the request, and closing idle connections",
+			Value: 5 * time.Minute,
+		},
+		cli.IntFlag{
+			Name:  "max-connections",
+			Usage: "[optional] Maximum number of simultaneous connections",
+			Value: 512,
+		},
+		cli.StringSliceFlag{
+			Name:  "http-white-list",
+			Usage: "[optional] White GET list for Prometheus HTTP request",
+			Value: &cli.StringSlice{
```

Comment:
Those URI should be hardcoded. They will be override if the users put the their white list here.